tool/src/java/org/sakaiproject/gradebookng/tool/panels/AddGradeItemPanelContent.java

```diff
@@ -22,24 +18,14 @@
 
 	private static final long serialVersionUID = 1L;
 
-	public AddGradeItemPanelContent(String id, Form form) {
-        this(id, form, "", "");
-	}
-
-    public AddGradeItemPanelContent(String id, Form form, String title, String points) {
+	public AddGradeItemPanelContent(String id) {
         super(id);
 
-        Assignment assignment = new Assignment();
-        assignment.setName(title);
-        if (points != null && !"".equals(points))
-            assignment.setPoints(Double.parseDouble(points));
-
-        add(new TextField("title", new PropertyModel<String>(assignment, "name")));
-        add(new TextField("points", new PropertyModel<Double>(assignment, "points")));
-        add(new DateTextField("duedate", new PropertyModel<Date>(assignment, "dueDate"), "MM/DD/yy"));
+        add(new TextField("name"));
```

Comment:
hey just checking you got this comment about using our own Assignment object (GbAssignment maybe) for the form, then copying over the values into the shared Assignment? Would be futureproofed a bit.